app/src/main/java/com/blogspot/e_kanivets/moneytracker/entity/RecordAdapterData.kt

```diff
@@ -0,0 +1,11 @@
+package com.blogspot.e_kanivets.moneytracker.entity
+
+import com.blogspot.e_kanivets.moneytracker.entity.data.Record
+
+sealed class RecordAdapterData
```

Comment:
Sealed classes are better like 
```
sealed class RecordItem { 
     data class Header(val date: String) : RecordItem()

     data class Record(val title: String, val categoryName: String, val fullPrice: Double, val currency: String, val isIncome: Boolean) : RecordAdapterData() {
    constructor(record: Record) : this(record.title, record.category?.name?.toString().orEmpty(), record.fullPrice, record.currency, record.isIncome)
}
 }
```

Then they can be referenced like `RecordItem.Header`, which is more obvious.